src/renderer/containers/TemplateEditorModal/index.tsx

```diff
@@ -54,6 +54,9 @@ const styles = require("./styles.pcss");
 
 const { Search } = Input;
 
+const ADD_ANNOTATION_DESCRIPTION = `Certain annotations trigger automated processes for new uploads.
+Optical Control ID and Is Optical Control are used to submit images for automatic alignment.
```

Comment:
Do users understand how they can find the aligned images?

Reply:
I think we don't have enough aligned images to really answer that question

Reply:
Do they get annotated in a way that links it to the original?  It seems like the ideal thing would be to give the original an annotation which links it to the derived image(s).